ExpressBindingGenerator/src/Meta/Schema.cpp

```diff
@@ -48,6 +52,14 @@ Entity Schema::getEntityByName(const std::string& name) const {
 	throw std::runtime_error("Entity does not exist.");
 }
 
+Entity& Schema::getEntityByName(const std::string& name) {
+	for ( auto& ent : entities_)
+		if (ent.getName() == name)
+			return ent;
+
+	throw std::runtime_error("Entity does not exist.");
+}

```

Comment:
This could be probably implemented with `find` function from standard library

Reply:
resolved in 7a9a70c